test/TensorFlow/no_copy.swift

```diff
@@ -1,4 +1,4 @@
-// RUN: %target-swift-frontend -Xllvm -tf-dump-intermediates -O -emit-sil -verify %s | %FileCheck %s
+// RUN: %target-swift-frontend -Xllvm -tf-dump-intermediates -Xllvm -tf-strict-deabstraction=false -O -emit-sil -verify %s | %FileCheck %s
```

Comment:
do you want to just remove the tests when strict DA is off, or do you prefer to leave that for a later PR?

Reply:
I will remove the tests for strict deabstraction in a subsequent PR.